src/thor/CMakeLists.txt

```diff
@@ -2,26 +2,27 @@ file(GLOB headers ${VALHALLA_SOURCE_DIR}/valhalla/thor/*.h)
 
 set(sources
   astar.cc
+  attributes_controller.cc
```

Comment:
i went to insert the new source file and saw this wasnt alphabetially sorted, so i did that